contentctl/objects/investigation.py

```diff
@@ -66,12 +64,18 @@ def serialize_model(self):
 
 
     def model_post_init(self, ctx:dict[str,Any]):
-        # director: Optional[DirectorOutputDto] = ctx.get("output_dto",None)
-        # if not isinstance(director,DirectorOutputDto):
-        #     raise ValueError("DirectorOutputDto was not passed in context of Detection model_post_init")
-        director: Optional[DirectorOutputDto] = ctx.get("output_dto",None)
+        # Ensure we link all stories this investigation references
+        # back to itself
         for story in self.tags.analytic_story:
             story.investigations.append(self)
     
+    def get_conf_stanza_name(self, app:CustomApp, max_stanza_length:int=81)->str:
```

Comment:
Again, because of duplication do we want a common Detection class that Baseline, Detection, and Investigation can inheritfrom?

Reply:
"Hopefully Someday"- this works for now, but I wouldn't rush to rewrite it. Especially given the number of validators we have on "search:" which seems to be the more in-common field.